osler/workup/models.py

```diff
@@ -101,9 +100,11 @@ class Meta:
         related_name="signed_%(app_label)s_%(class)s")
     signed_date = models.DateTimeField(blank=True, null=True)
 
+    class Meta:
```

Comment:
any reason why the class Meta is down here? stylistically I prefer at the beginning

Reply:
in the Django source code, they always put it in between fields and methods, so I was following their style

Reply:
Ok sounds good then!